src/PageTypes/SitemapPage.php

```diff
@@ -8,7 +22,7 @@ class SitemapPage extends Page {
 	private static $plural_name = 'Sitemap Pages';
 
 }
-class SitemapPage_Controller extends Page_Controller {
+class SitemapPage_Controller extends PageController {
```

Comment:
Underscore shouldn't be a thing in the class name, not just the extension.

Also it should be in another file.